docs/datafeed.md

```diff
@@ -0,0 +1,127 @@
+# Datafeed
+
+The Datafeed Service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
+The bot can create a datafeed, list all created datafeeds or retrieve all the Real Time events within a datafeed through Datafeed API.
+The Datafeed Service is a Core service built on top of the Datafeed API and provide a dedicated contract to bot developers to work with datafeed. 
```

Comment:
I'd also mention (somewhere, as a separate paragraph) that the [Activity API](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/docs/activity-api.md) is built on top of the datafeed.

Reply:
I thought it's already mentioned in the documentation of Activity API?

Reply:
It's not forbidden to have bidirectional linking in our documentation, it's even better

Reply:
Did we already introduced the concept of Core service in the documentation?

Reply:
Not yet, maybe we should have something here : https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/docs/index.md?